src/routes/transactions/mappers/common/data-decoded-param.helper.spec.ts

```diff
@@ -400,4 +400,223 @@ describe('DataDecoded param helper (Unit)', () => {
       expect(fromParam).toBe('fallback');
     });
   });
+
+  describe('hasNestedDelegate function tests', () => {
```

Comment:
nitpick: remove "function tests" from the name. It's too specific to the "implementation" in my opinion. Can be just `hasNestedDelegate` or `hasNestedDelegate()`

Reply:
[a158c2a](https://github.com/5afe/safe-client-gateway-nest/pull/291/commits/a158c2a02ceed6346e496ef948aac0185e20225d)